packages/@dcl/sdk-commands/src/commands/start/data-layer/engine-state.json

```
@@ -0,0 +1,99 @@
+// TODO: Remove this file. It's just the editor sample
+{
+  "512": {
+    "1": {
```

maybe use "componentName" instead "componentId"
```suggestion
    "core::Transform": {
```